components/centraldashboard/package-lock.json

```diff
@@ -8673,7 +8673,8 @@
                         "ansi-regex": {
                             "version": "2.1.1",
                             "bundled": true,
-                            "dev": true
+                            "dev": true,
+                            "optional": true
```

Comment:
Where is all the `"optional": true` coming from? I wouldn't expect this PR to require dependency updates.